src/herbivoresAndCarnivores.js

```diff
@@ -2,16 +2,46 @@
 
 class Animal {
   // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+  }
 }
 
 class Herbivore extends Animal {
   // write your code here
+  constructor(name, health) {
+    super(name, health);
+
+    this.hidden = false;
+    Animal.alive.push(this);
+  }
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
   // write your code here
+  constructor(name, health) {
+    super(name, health);
+
+    Animal.alive.push(this);
+  }
+
+  bite(herbi) {
```

Comment:
let's give this one a more meaningful name
```suggestion
  bite(victim) {
```